dotnetv3/Glue/README.md

```diff
@@ -0,0 +1,73 @@
+# AWS Glue code examples for the AWS SDK for .NET v3
```

Comment:
AWS SDK for .NET v3 --> SDK for .NET (v3)

Reply:
changed